pkgs/top-level/all-packages.nix

```diff
@@ -25791,6 +25791,14 @@ in
 
   octopus = callPackage ../applications/science/chemistry/octopus { };
 
+  openlp = libsForQt514.callPackage ../applications/misc/openlp { };
+  openlpFull = openlp.override {
+    pdfSupport = true;
```

Comment:
Can we remove all options if they are on by default anyway? I think they don't get probably tested in all combinations and it would make the package a bit easier.

Reply:
In `openlp` as opposed to `openlpFull` they are turned off, and the difference in closure size is around 1.5 GB. Therefore it makes sense to me to have a minimal package and a full one. I ususally don't use the extra features, so for the basic use case (only music slides) the minimal package is sufficient.

I could unify the options such that they are jointly turned on and off. Would this be better for being simpler to test?
In principle they should be almost orthogonal (not interfering with each other), since they turn on probably independent runtime features by simply including the necessary dependencies in the closure. The only interference I know of is between `enableVlc` and `gstreamerSupport` in the discovery of different audio/video players, which can be selected at runtime. Of course I have no proof of non-interference of the rest.

I tested the feature sets of `openlp` and `openlpFull` but no other configurations.

Reply:
1.5GB is quite a bit. I think it is worth keeping it. Thanks for explaining.